ui/src/app/layout/topics/schemasection/schema-section.component.html

```diff
@@ -53,6 +53,12 @@
                             <input [(ngModel)]="schemaChangeDescription" type="text" size="50"
                                    class="form-control"/><br>
                         </div>
+                        <div *ngIf="isAdmin" class="checkbox">
+                            <label>
+                                <input type="checkbox"
+                                       [(ngModel)]="skipCompatCheck"> {{ 'Skip compatibility check' | translate }}
```

Comment:
An `ngb-alert` with warning type should be included here and state, "**Caution**: This allows for incompatible changes in the topic data. Only use this if you really know what you are doing."

The alert should only be displayed when the checkbox is checked.